airbyte-integrations/connectors/destination-redshift/src/test-integration/java/io/airbyte/integrations/destination/redshift/RedshiftInsertDestinationAcceptanceTest.java

```diff
@@ -4,167 +4,20 @@
 
 package io.airbyte.integrations.destination.redshift;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-
 import com.fasterxml.jackson.databind.JsonNode;
-import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.Lists;
 import io.airbyte.commons.json.Jsons;
-import io.airbyte.commons.string.Strings;
-import io.airbyte.db.Database;
-import io.airbyte.integrations.base.AirbyteMessageConsumer;
-import io.airbyte.integrations.base.Destination;
-import io.airbyte.integrations.base.JavaBaseConstants;
-import io.airbyte.protocol.models.JsonSchemaType;
-import io.airbyte.protocol.models.v0.AirbyteMessage;
-import io.airbyte.protocol.models.v0.AirbyteRecordMessage;
-import io.airbyte.protocol.models.v0.AirbyteStateMessage;
-import io.airbyte.protocol.models.v0.CatalogHelpers;
-import io.airbyte.protocol.models.v0.ConfiguredAirbyteCatalog;
-import io.airbyte.protocol.models.v0.DestinationSyncMode;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.sql.SQLException;
-import java.time.Instant;
-import java.util.List;
-import org.apache.commons.lang3.RandomStringUtils;
-import org.jooq.Record;
-import org.jooq.Result;
-import org.junit.jupiter.api.Disabled;
-import org.junit.jupiter.api.Test;
 
 /**
  * Integration test testing the {@link RedshiftInsertDestination}. As the Redshift test credentials
  * contain S3 credentials by default, we remove these credentials.
  */
 public class RedshiftInsertDestinationAcceptanceTest extends RedshiftStagingS3DestinationAcceptanceTest {
```

Comment:
Do we still need this file and `RedshiftS3StagingInsertDestinationAcceptanceTest.java` below? There are no tests inside anymore.

Reply:
Yes, because they will execute the tests using the `config.json` tests for `RedshiftInsertDestinationAcceptanceTest`, as for `RedshiftS3StagingInsertDestinationAcceptanceTest` that doesn't seem necessary since the same config of `config_staging.json` will be used within `RedshiftStagingS3Destination`

/rant `RedshiftInsert` should not be extending from `RedshiftStagingS3`

Reply:
🤷‍♀️ ok I'll leave `RedshiftS3StagingInsertDestinationAcceptanceTest` to mirror `RedshiftInsertDestinationAcceptanceTest` to avoid confusion why there's one and not the other.